pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -105,17 +105,14 @@ var (
 )
 
 func Test_successful_parsing_of_addon_dir_without_remove_addon_file(t *testing.T) {
-	path := filepath.Join(basepath, "..", "..", "..", "..", "addons", "anyuid")
+	path := filepath.Join(basepath, "..", "..", "..", "..", "addons", "xpaas")
 	addOn, err := testParser.Parse(path)
 
 	assert.NoError(t, err, "Error in parsing addon content")
 
-	expectedName := "anyuid"
+	expectedName := "xpaas"
 	assert.Equal(t, expectedName, addOn.MetaData().Name())
 
-	expectedNumberOfCommands := 5
-	assert.Len(t, addOn.Commands(), expectedNumberOfCommands)
-
```

Comment:
I am removing this because everytime an addon modified then the number of commands might increase or decrease this test purpose is making sure that an addon without having a remove file should be parsed and have info.

Reply:
+1